tests/csapi/account_data_test.go

```diff
@@ -0,0 +1,83 @@
+package csapi_tests
+
+import (
+	"testing"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+)
+
+func TestAddAccountData(t *testing.T) {
+	deployment := Deploy(t, b.BlueprintOneToOneRoom)
+	defer deployment.Destroy(t)
+
+	alice := deployment.Client(t, "hs1", "@alice:hs1")
+
+	// sytest: Can add account data
+	// sytest: Can get account data without syncing
+	t.Run("Can add global account data", func(t *testing.T) {
+		// Set the account data entry
+		resp := alice.SetGlobalAccountData(t, "com.beeper.test", map[string]interface{}{"value": "first"})
```

Comment:
Speaking with my SCT hat on; it feels a little off to put a company's identifier into Complement for anything other than an unstable prefix. It comes across as a bit of an advertisement?